src/containers/profile/Profile.jsx

```diff
@@ -0,0 +1,103 @@
+import React from "react";
+import { connect } from "react-redux";
+import { firestoreConnect } from "react-redux-firebase";
+import { compose } from "redux";
+import { Redirect } from "react-router-dom";
+import { Tabs, Tab, Container, Row, Col } from "react-bootstrap";
+import anonymousImage from "../../images/anonymousImage.png";
+import profileHeaderBG from "../../images/profileHeaderBG.png";
+import Header from "../../components/common/Header";
+import "./Profile.css";
+import AccountSettings from "./ProfileSections/AccountSettings/AccountSettings";
+import ProfilesList from "./ProfileSections/ProfilesList/ProfilesList";
+
+const Profile = (props) => {
+  const { profile, auth, id, isblocked } = props;
+
+  const extraComponents = () => {
+    return (
+      <div className="extraComponents">
```

Comment:
#60 also seems to contain this file so i think you may want to double check you split your PRs correctly

https://github.com/ReCoded-Org/istanbul-capstone-sdp/pull/60/files

it's usually easier to plan the splits from the start than to write and split them later :)